math/pi/montecarlopi_test.go

```diff
@@ -0,0 +1,27 @@
+// montecarlopi_test.go
+// description: Test for calculating pi by the Monte Carlo method
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see montecarlopi.go
+
+package pi
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestMonteCarloPi(t *testing.T) {
+	t.Run("BruteForce Pi", func(t *testing.T) {
```

Comment:
Why is this test named as BruteForce Pi?

Reply:
Fix